### PR TITLE
Added a scheme for contacts added/removed from the list of contacts

### PR DESCRIPTION
**Pull request by strangepleasures**

### CODE REVIEW

File: commons/phone/phone_contacts_list.avsc [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "namespace": "org.radarcns.phone",
+  "type": "record",
+  "name": "PhoneContactsList",
+  "doc": "Number of new contacts added/removed from the list of contacts between this and the previous record.",
+  "fields": [
+    {"name": "time", "type": "double", "doc": "device timestamp in UTC (s)"},
+    {"name": "timeReceived", "type": "double", "doc": "device receiver timestamp in UTC (s)"},
+    {"name": "contactsAdded", "type": "int", "doc": "Number of new contacts added to the list of contacts."},
```
Comment (blootsvoets) [inline commons/phone/phone_contacts_list.avsc]: perhaps make the type `["null", "int"]`: the first time you scan this number is unknown? To the doc, add something like

> Number of new contacts added to the list of contacts _since the previous measurement that was sent. The value is null if this is the first measurement or if the previous measurement is unknown to the sender._

Same for `contactsRemoved`.

File: commons/phone/phone_contacts_list.avsc [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    {"name": "contactsRemoved", "type": "int", "doc": "Number of new contacts removed from the list of contacts."},
+    {"name": "contactsNumber", "type": "int", "doc": "Current number of contacts in the list of contacts."}
```
Comment (blootsvoets) [inline commons/phone/phone_contacts_list.avsc]: rename to just `contacts`?

File: commons/phone/phone_contacts_list.avsc [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+  "namespace": "org.radarcns.phone",
+  "type": "record",
+  "name": "PhoneContactsList",
```
Comment (blootsvoets) [inline commons/phone/phone_contacts_list.avsc]: This should probably be called `PhoneContactList`. Also see the following comment.

File: commons/phone/phone_contacts_list.avsc [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  "doc": "Number of new contacts added/removed from the list of contacts between this and the previous record.",
```
Comment (blootsvoets) [inline commons/phone/phone_contacts_list.avsc]: Either we make this type more specific, like `PhoneContactListModification` or we add a more generic description here.

File: commons/phone/phone_contacts_list.avsc (hunk 1 above)
Comment (strangepleasures) [reply commons/phone/phone_contacts_list.avsc]: Done

File: commons/phone/phone_contacts_list.avsc (hunk 4 above)
Comment (strangepleasures) [reply commons/phone/phone_contacts_list.avsc]: Added a better description

File: commons/phone/phone_contacts_list.avsc (hunk 3 above)
Comment (strangepleasures) [reply commons/phone/phone_contacts_list.avsc]: Renamed

File: commons/phone/phone_contacts_list.avsc (hunk 2 above)
Comment (strangepleasures) [reply commons/phone/phone_contacts_list.avsc]: Done

File: commons/phone/phone_contact_list.avsc [hunk 5]
```diff
@@ -0,0 +1,13 @@
+{
+  "namespace": "org.radarcns.phone",
+  "type": "record",
+  "name": "PhoneContactList",
+  "doc": "Data describing the number of contacts in the the list of contacts and its change since the previous check.",
+  "fields": [
+    {"name": "time", "type": "double", "doc": "device timestamp in UTC (s)"},
+    {"name": "timeReceived", "type": "double", "doc": "device receiver timestamp in UTC (s)"},
+    {"name": "contactsAdded", "type": ["null", "int"], "doc": "Number of new contacts added to the list of contacts since the previous measurement that was sent. The value is null if this is the first measurement or if the previous measurement is unknown to the sender."},
+    {"name": "contactsRemoved", "type": ["null", "int"], "doc": "Number of new contacts removed from the list of contacts since the previous measurement that was sent. The value is null if this is the first measurement or if the previous measurement is unknown to the sender."},
```
Comment (blootsvoets) [inline commons/phone/phone_contact_list.avsc]: Could you please add `"default": null` to `contactsAdded` and `contactsRemoved`?

Comment (strangepleasures) [reply commons/phone/phone_contact_list.avsc]: Done

**Pull request merged**